README.md

```diff
@@ -0,0 +1,49 @@
+# URLess
+
+[![GitHub license](https://img.shields.io/github/license/Naereen/StrapDown.js.svg)](https://github.com/Naereen/StrapDown.js/blob/master/LICENSE)
+
+Shorten (or slightly lengthen until we get a shorter domain) your URLs in one easy click!
+
+## Installation & Usage
+
+Visit [URLess](https://urlessen.herokuapp.com/)!
+
+Or clone/download the repo and:
+
+* Open a terminal and navigate into the folder
+* Run `pipenv install -r requirements.txt` to install dependencies
+* Navigate into /shortener/ and run `./manage.py runserver` to start locally
+
+## Technologies
+* Python
+* Django with templating
+* SQLite
+* Heroku and PostgreSQL for deployment
+* A little bit of Javascript
+
+## Process
+
+* Created base django project and urls app
+* Added basic views
+* Created URL model, form and view
+* Migrated model to database and tested saving/retrieving
+* Added generation of short url and display on view as message
+* Added styling
+
+## Wins & Challenges
+
+### Wins
+
+* Managed to deploy!
+
+### Challenges
+
+* Deployment
+
```

Comment:
It's nice you mention this, but some specifics would also be appreciated!  What made deployment difficult, and how did you overcome this challenge? (Although I appreciate you may have run out of time to describe this properly given how busy this week has been and how polished your app is!)

Reply:
yeah wanted to put more but it would take some time - essentially there's a lot of configuration and extra modules needed to set it up (for heroku at least) - there's a good [MDN guide](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/Deployment) we followed that worked in the end